.cbmc-batch/jobs/Makefile.common

```diff
@@ -149,7 +149,7 @@ common-git:
 	@if [ ! -d $(COMMONDIR) ] ;\
 	then \
 		cd $(BASEDIR); \
-		git clone --quiet --depth 30 https://github.com/awslabs/aws-c-common.git $(COMMON_REPO_NAME); \
+		git clone --quiet --depth 100 https://github.com/awslabs/aws-c-common.git $(COMMON_REPO_NAME); \
```

Comment:
I suggest removing `--depth` entirely, since this problem will just recur again in the future.

Reply:
Great, it's been removed!